project_setup.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# test script to make a project folder with a blank readme
+# Emma Gibson egibson@usfca.edu
+# August 30, 2017
+
+# dynamic dates: mkdir $(date "+%Y-%m-%d)_bioinformatics_project
+
+# make project folders
+echo "creating folder structure..."
+mkdir $(date "+%Y-%m-%d")_bioinformatics_project
+cd 2017-08-30_bioinformatics_project
+mkdir data data/raw_data code output output/tables output/figures
+echo "created folder structure"
+
+#to download data
+echo "downloading raw data..."
+cd data/raw_data
```

Comment:
You could also use `curl` to download the file to this directory directly instead of using `cd`.